crates/swift-bridge-ir/src/bridged_type/bridgeable_result.rs

```diff
@@ -253,6 +255,18 @@ impl BuiltInResult {
                 ),
                 TypePosition::SharedStructField => todo!(),
                 TypePosition::SwiftCallsRustAsyncOnCompleteReturnTy => todo!(),
+                TypePosition::ThrowingInit(lang) => {
+                    match lang {
+                        HostLang::Rust => format!(
+                            "let val = {expression}; switch val.tag {{ case {c_ok_name}: self.init(ptr: val.payload.ok) case {c_err_name}: throw {err_swift_type} default: fatalError() }}",
```

Comment:
I guess that it's not suitable because the `convert_ffi_value_to_swift_value` method returns a expression.

Reply:
I'm not sure what you're communicating?

Reply:
I think you're asking whether it's okay to return two statements here instead of a single expression?

This should be fine since the `ThrowingInit` code here is only ever used inside of an initializer and isn't ever embedded inside of another expression.

Reply:
The other return values of `convert_ffi_value_to_swift_value` return closures, but this case doesn't	return a closure. This means that returns a simple statement, not expression.

For examples:
https://github.com/chinedufn/swift-bridge/blob/495611ba39bc6b08838408751b92f6329b98f321/crates/swift-bridge-ir/src/bridged_type/bridgeable_result.rs#L286

https://github.com/chinedufn/swift-bridge/blob/495611ba39bc6b08838408751b92f6329b98f321/crates/swift-bridge-ir/src/bridged_type/bridgeable_result.rs#L247

The above examples return closures.